doc/policy_tutorial/DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,94 @@
+# Setting up the development environment
+
+As was clear from the introduction, APIcast policies are created in the Lua programming language. So we need to setup an environment to do some Lua programming. Also, an actual APIcast server would be very nice to perform some local tests.
+
+Luckily APIcast makes it very easy to setup a development environment for APIcast using Docker and Docker Compose.
+
+### prerequisites
+This means both Docker and Docker compose must be installed.
+
+The version of Docker I currently use is:
+
+    Docker version 19.03.8
+
+Instructions for installing Docker can be found on the Docker [website](https://docs.docker.com/get-docker/).
+
+With Docker compose version:
+
+    docker-compose version 1.25.4
+
+Instructions for installing Docker-compose can also be found on the Docker [website](https://docs.docker.com/compose/install/).
+
+### Setting up the development image
+Now that we have both Docker and Docker-compose installed we an setup the APIcast development image.
+
+Firstly the APIcast git repostitory must be cloned so we can start the development of our policy. Since we are going to base our policy on the latest 3scale release we are switching to the stable branch of APIcast.
+
+```shell
+git clone https://github.com/3scale/apicast.git
+```
+
+when done switch to a stable branch, I am using 3.8
+```shell
+cd apicast/
+git checkout 3.8-stable
+```
+
+To start the APIcast containers using Docker-compose we can use the Make file provided by 3scale. In the APIcast directory simply execute the command:
+```shell
+make development
+```
+
+![make-development](img/make-development.png)
```

Comment:
Is this image essential to provide a clear understanding of the configuration process? 

- If `yes`, please include a paragraph explaining the image.
- If `no`, you can remove the image.

Reply:
leaving the image. It gives a clear understanding of what it does. For those familiar with docker and docker-compose (which is a requisite for any APIcast contributor)

I would not add an image, instead I would copy those lines in a code block. But good enough

Reply:
replace the image with the text